### PR TITLE
feature: UI Cleanup, Play button changes, Device auto-reloading

### LINKED ISSUE #22: General UI Cleanup
Switching over to styled-components messed up some of the spacing/margins/etc a bit, will fix. Also going to do some touch ups and make things look a little nicer

### PR DESCRIPTION
**Pull request by dwilliams27**
- Cleaned up some small things in UI
- Changed playbutton to be on each playlist, instead of top right
- Device list will auto-reload when clicking on it, no need to refresh page anymore
- Switched js files to ts

Closes #22 

image

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+html {
+  height: 100%;
+}
+
 /* Gradient doesnt work with styled-components */
```
Comment (ColemanMitch) [inline src/styles/main.css:6]: Not sure if this is doing anything?

Comment (ColemanMitch) [reply src/styles/main.css:6]: Note the white space at the bottom when I filter for something that'll turn up no playlist matches

image

Comment (dwilliams27) [reply src/styles/main.css:6]: Tried for like 15 mins to fix this, no clue why this isnt working. Supposedly setting the html to 100% height and then body to min-height 100% is the way to go, which I did and it looked like the body encompassed the whole page. But then the root div even when set to min-height 100%, wouldnt fill. Made me mad so I just left as is lol, I'll add a TODO to look into later

Comment (ColemanMitch) [reply src/styles/main.css:6]: Bro I did the same thing and found myself spinning in circles. Agree with leaving it as a TODO for now

**Pull request merged**